charts/flyte-core/templates/_helpers.tpl

```diff
@@ -35,6 +35,45 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 {{- end -}}
 
+{{- define "flyteadmin.service.grpc.name" -}}
+{{- if .Values.common.ingress.separateGrpcIngress }}
+{{- printf "%s-grpc" ( include "flyteadmin.name" . ) -}}
+{{- else }}
+{{- template "flyteadmin.name" . -}}
+{{- end}}
+{{- end -}}
+
+{{- define "flyteadmin.service.grpc.port" -}}
+{{- if eq .Values.configmap.adminServer.server.security.secure true -}}
+80
+{{- else -}}
+81
+{{- end }}
+{{- end -}}
+
+{{- define "flyteadmin.service.http.port" -}}
+80
```

Comment:
What if someone needs to override this? I guess putting it into a template would complicate it